packages/swirl-components/src/components/swirl-pdf-reader/swirl-pdf-reader.css

```diff
@@ -1,6 +1,8 @@
 @import "../../styles/media-queries.css";
 
 :host {
+  --thumbnails-width: 6rem;
```

Comment:
(issue): in the design the thumbnail container is a little bit bigger. with 112 px which should be 7.

Reply:
Fixed.